src/components/feedback.vue

```diff
@@ -140,11 +140,11 @@ export default {
   computed: {
     droits() {
       const droitId = this.route.params.droitId
-      const droits = droitId
-        ? this.store.calculs.resultats.droitsEligibles.filter(
-            (droit) => droit.id === this.route.params.droitId
-          )
-        : this.store.calculs.resultats.droitsEligibles
+      const droits = this.store.calculs.resultats.droitsEligibles
+        ? this.store.calculs.resultats.droitsEligibles.filter((droit) => {
+            return !droitId || droit.id === droitId
+          })
+        : []
```

Comment:
Je trouve cette condition `!droitId || droit.id === droitId` compliquée à comprendre.


```suggestion
      const droits = droitId
          ? [this.store.calculs.resultats?.droitsEligibles?.find((droit) => {
            return droit.id === droitId
          })]
        : this.store.calculs.resultats.droitsEligibles
```